<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectTypesEditor</class>
 <widget class="QMainWindow" name="ObjectTypesEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Object Types Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="objectTypesTable">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="addObjectTypeButton">
            <property name="toolTip">
             <string>Add Object Type</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
            <property name="icon">
             <iconset resource="tiled.qrc">
              <normaloff>:/images/22x22/add.png</normaloff>:/images/22x22/add.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="removeObjectTypeButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Remove Selected Object Types</string>
            </property>
            <property name="text">
             <string>Remove</string>
            </property>
            <property name="icon">
             <iconset resource="tiled.qrc">
              <normaloff>:/images/22x22/remove.png</normaloff>:/images/22x22/remove.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QtTreePropertyBrowser" name="propertiesView" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="addPropertyButton">
            <property name="toolTip">
             <string>Add Object Type</string>
            </property>
            <property name="text">
             <string>Add Property</string>
            </property>
            <property name="icon">
             <iconset resource="tiled.qrc">
              <normaloff>:/images/22x22/add.png</normaloff>:/images/22x22/add.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="removePropertyButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Remove Selected Object Types</string>
            </property>
            <property name="text">
             <string>Remove Property</string>
            </property>
            <property name="icon">
             <iconset resource="tiled.qrc">
              <normaloff>:/images/22x22/remove.png</normaloff>:/images/22x22/remove.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="renamePropertyButton">
            <property name="text">
             <string>Rename Property</string>
            </property>
            <property name="icon">
             <iconset resource="tiled.qrc">
              <normaloff>:/images/16x16/rename.png</normaloff>:/images/16x16/rename.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionChooseFile"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport">
   <property name="text">
    <string>Export Object Types...</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import Object Types...</string>
   </property>
  </action>
  <action name="actionChooseFile">
   <property name="text">
    <string>Choose Object Types File...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QWidget</extends>
   <header location="global">QtTreePropertyBrowser</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="tiled.qrc"/>
 </resources>
 <connections/>
</ui>
